desktop/ui/pom.xml

```diff
@@ -257,6 +258,11 @@
 			<groupId>org.apache.httpcomponents</groupId>
 			<artifactId>httpmime</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.freemarker</groupId>
+			<artifactId>freemarker</artifactId>
+			<version>2.3.21</version>
```

Comment:
Let's have some management of the version by making freemarker a managed dependency in the parent pom. Then we also should not specify the version in the monitor module(s).